torch/fft/__init__.py

```diff
@@ -9,18 +9,283 @@
+    Hermitian property: `X[i] = conj(X[-i])`. This function always returns both
+    the positive and negative frequency terms, even though for real inputs the
+    negative frequencies are redundant. :func:`~torch.fft.rfft` returns the more
+    efficient "half-complex" representation where only the positive frequencies
+    are stored explicitly.
+
 Args:
-  input (Tensor): the input tensor
+    input (Tensor): the input tensor
+    n (int, optional): Signal length. If given, the input will either be zero-padded
+        or trimmed to this length before taking the FFT.
+    dim (int, optional): The dimension along which to take the one dimensional FFT.
+    norm (str, optional): Normalization mode. Between the forward and backward
+        transforms (:func:`~torch.fft.fft` and :func:`~torch.fft.ifft`) a total
+        normalization factor of ``1/n`` is applied. The mode determines where
+        this normalization is applied:
+
+        * ``"forward"`` - the forward transform is normalized by ``1/n``, backward is unnormalized.
+        * ``"backward"`` - the backward transform is normalized by ``1/n``, forward is unnormalized.
+        * ``"ortho"`` - both transforms are normalized by ``1/sqrt(n)`` (making it orthonormal).
 
-Example::
+        Default is ``"backward"``.
 
-    >>> t = torch.randn(4, dtype=torch.complex128)
+Example:
+
+    >>> import torch.fft
+    >>> t = torch.arange(4)
     >>> t
-    tensor([-1.1364-0.5694j, -0.6637+0.9987j, -1.0102-0.4383j,  0.3017+0.9371j], dtype=torch.complex128)
+    tensor([0, 1, 2, 3])
+    >>> torch.fft.fft(t)
+    tensor([ 6.+0.j, -2.+2.j, -2.+0.j, -2.-2.j])
+
+    >>> t = tensor([0.+1.j, 2.+3.j, 4.+5.j, 6.+7.j])
     >>> torch.fft.fft(t)
-    tensor([-2.5086+0.9281j, -0.0646+0.8343j, -1.7846-2.9435j, -0.1878-1.0965j], dtype=torch.complex128)
+    tensor([12.+16.j, -8.+0.j, -4.-4.j,  0.-8.j])
+""")
+
+ifft = _add_docstr(_fft.fft_ifft, r"""
+ifft(input, n=None, dim=-1, norm=None) -> Tensor
+
+Computes the one dimensional inverse discrete Fourier transform of :attr:`input`.
+
+Args:
+    input (Tensor): the input tensor
+    n (int, optional): Signal length. If given, the input will either be zero-padded
+        or trimmed to this length before taking the IFFT.
+    dim (int, optional): The dimension along which to take the one dimensional IFFT.
+    norm (str, optional): Normalization mode. Between the forward and backward
+        transforms (:func:`~torch.fft.fft` and :func:`~torch.fft.ifft`) a total
+        normalization factor of ``1/n`` is applied. The mode determines where
+        this normalization is applied:
+
+        * ``"forward"`` - the forward transform is normalized by ``1/n``, backward is unnormalized.
+        * ``"backward"`` - the backward transform is normalized by ``1/n``, forward is unnormalized.
+        * ``"ortho"`` - both transforms are normalized by ``1/sqrt(n)`` (making it orthonormal).
+
+        Default is ``"backward"``.
```

Comment:
nit: Default is backward (normalize by 1/n). 

If you like the suggestion above about how to explain these modes you could also point to the documentation for fft to explain the normalization mode relationship instead of duplicating the description.